nginx.conf

```diff
@@ -13,13 +13,34 @@
 #
 # One difference between this file and the configuration for production is that
 # this configuration does not support request body decompression.
+daemon off;
+error_log stderr;
+pid ./.nginx/nginx.pid;
 
 events {
 }
 http {
-  # This path may be different on your local computer depending on your OS and
-  # how you installed nginx.
-  include /usr/local/etc/nginx/mime.types;
+  access_log ./.nginx/nginx-access.log;
+
+  types {
```

Comment:
What are the implications of this? Like if I have a jpg or mov or m4v or any other file type that's not explicitly in this list specified as a form or submission attachment, what happens?

Reply:
My understanding is that the content-type header for attachments is set in backend code.

Reply:
https://github.com/getodk/central-backend/blob/master/lib/resources/submissions.js#L397 -> https://github.com/getodk/central-backend/blob/master/lib/util/http.js#L100

tests at: https://github.com/getodk/central-backend/blob/master/test/integration/api/submissions.js#L3517 etc